lib/puppet/indirector/file_server.rb

```diff
@@ -15,7 +15,15 @@ def authorized?(request)
 
     # If we're not serving this mount, then access is denied.
     return false unless mount
-    mount.allowed?(request.node, request.ip)
+
+    # If there are no auth directives or there is an 'allow *' directive, then
+    # access is allowed.
+    if mount.empty? || mount.globalallow?
```

Comment:
@ahpook weighed in the question about moving to default allow instead of default deny on the JIRA ticket [here](https://tickets.puppetlabs.com/browse/PUP-6359?focusedCommentId=450960&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-450960).  He's in agreement with the idea of moving to default allow since it enables users to avoid having to have any security directives in a fileserver mount point definition, which is ultimately what we'd like to see happen.  He also said he convinced @MosesMendoza in an offline conversation to go with this as well.  Given that, I'm going to leave the implementation as-is on the PR.